x/axelarnet/types/params.go

```diff
@@ -46,39 +43,35 @@ func (m *Params) ParamSetPairs() params.ParamSetPairs {
 		set on the correct Params data struct
 	*/
 	return params.ParamSetPairs{
-		params.NewParamSetPair(KeyAssets, &m.SupportedChains, validateSupportedChains),
-		params.NewParamSetPair(KeyRouteTimeoutWindow, &m.RouteTimeoutWindow, validateUint64("RouteTimeoutWindow")),
+		params.NewParamSetPair(KeyRouteTimeoutWindow, &m.RouteTimeoutWindow, validatePosUInt64("RouteTimeoutWindow")),
 		params.NewParamSetPair(KeyTransactionFeeRate, &m.TransactionFeeRate, validateTransactionFeeRate),
 	}
 }
 
 // Validate checks if the parameters are valid
 func (m Params) Validate() error {
-	return validateSupportedChains(m.SupportedChains)
-}
-
-func validateSupportedChains(infos interface{}) error {
-	supportedChains, ok := infos.([]string)
-	if !ok {
-		return sdkerrors.Wrapf(types.ErrInvalidGenesis, "invalid parameter type for %T: %T", []string{}, infos)
+	if err := validatePosUInt64("RouteTimeoutWindow")(m.RouteTimeoutWindow); err != nil {
+		return err
 	}
 
-	for _, chain := range supportedChains {
-		if chain == "" {
-			return sdkerrors.Wrap(types.ErrInvalidGenesis, "chain name cannot be an empty string")
-		}
+	if err := validateTransactionFeeRate(m.TransactionFeeRate); err != nil {
+		return err
 	}
 
 	return nil
 }
 
-func validateUint64(field string) func(value interface{}) error {
+func validatePosUInt64(field string) func(value interface{}) error {
 	return func(value interface{}) error {
-		_, ok := value.(uint64)
+		val, ok := value.(uint64)
 		if !ok {
 			return fmt.Errorf("invalid parameter type for %s: %T", field, value)
 		}
 
+		if val <= 0 {
```

Comment:
```suggestion
		if val == 0 {
```
`val` cannot be be <0, so this check is unnecessarily confusing

Reply:
Yeah, I kept it thinking if this gets switched to int64 for some reason. I'll use `==`.